### DISCUSSION TITLE
Refund

### DISCUSSION THREAD

**Discussion by alexanderalvarado91** [vercel/community#6178] [Account Issue] [answered]
### Summary

I've upgraded my product but it seems like it's not necessary for now. I've already downgraded but can I get a refund please?

### Steps to Reproduce

Check my account.

**Comment by amyegan** [comment, marked as answer]
Hey @alexanderalvarado91. Please [submit a case](https://vercel.com/help) to the Customer Success Team. That will get you in touch with the billing experts who can review your account details. 🙏

**Reply by alexanderalvarado91** [reply]
@amyegan I've downgraded to the Hobby Plan so I don't see the support tab.

**Reply by amyegan** [reply]
You don't need the support tab to submit a case. Just go to https://vercel.com/help and click "Get Help" to open the support for. That will let you share info about your situation and create a case. 🙂
